client/a8c-for-agencies/sections/referrals/primary/referrals-overview/index.tsx

```diff
@@ -116,7 +122,7 @@ export default function ReferralsOverview( {
 
 					<LayoutHeader>
 						<Title>{ title } </Title>
-						{ isAutomatedReferral && (
+						{ isAutomatedReferral && isAgencyApproved && (
```

Comment:
We should still have the <MobileSidebarNavigation /> here. We should only hide the button